web/thesauruses/python/3/lists.json

```diff
@@ -0,0 +1,492 @@
+{
+  "meta": {
+    "language": "python",
+    "language_name": "Python",
+    "structure": "lists",
+    "language_version": "3"
+  },
+  "concepts": {
+    "name_of_ordered_mutable_list": {
+      "code": [
+        "list"
+      ]
+    },
+    "create_a_ordered_mutable_list": {
+      "code": [
+        "list_name = []",
+        "list_name = [value]",
+        "list_name = [value1, value2, ...]"
+      ],
+      "comment": "First line creates an empty list. Second line creates a list with one value. Third line creates a list with multiple values"
+    },
+    "ordered_mutable_list_start_number": {
+      "code": [
+        "0"
+      ]
+    },
+    "ordered_mutable_list_can_be_appended": {
+      "code": [
+        "Yes"
+      ]
+    },
+    "ordered_mutable_list_can_be_inserted_in_middle": {
+      "code": [
+        "Yes"
+      ]
+    },
+    "access_element_in_ordered_mutable_list": {
+      "code": [
+        "list_name[index]"
+      ]
+    },
+    "insert_into_beginning_of_ordered_mutable_list": {
+      "code": [
+        "list_name.insert(0, value)"
+      ]
+    },
+    "insert_into_end_of_ordered_mutable_list": {
+      "code": [
+        "list_name.append(value)"
+      ]
+    },
+    "insert_into_middle_of_ordered_mutable_list": {
+      "code": [
+        "list_name.insert(index, value)"
+      ]
+    },
+    "erase_element_at_beginning_of_ordered_mutable_list": {
+      "code": [
+        "list_name.pop(0)"
+      ]
+    },
+    "erase_element_at_end_of_ordered_mutable_list": {
+      "code": [
+        "list_name.pop()"
+      ]
+    },
+    "erase_element_in_middle_of_ordered_mutable_list": {
+      "code": [
+        "list_name.pop(index)",
+        "list_name.remove(value)"
+      ],
+      "comment": "First line removes element by index. Second line removes element by value"
+    },
+    "swap_elements_in_ordered_mutable_list": {
+      "code": [
+        "list_name[index1], list_name[index2] = list_name[index2], list_name[index1]"
+      ]
+    },
+    "delete_ordered_mutable_list": {
+      "code": [
+        "del list_name"
+      ]
+    },
+    "name_of_unordered_mutable_list": {
+      "code": [
+        "set"
+      ]
+    },
+    "create_a_unordered_mutable_list": {
+      "code": [
+        "set_name = {}",
+        "set_name = {value}",
+        "set_name = {value1, value2, ...}"
+      ],
+      "comment": "First line creates an empty set. Second line creates a set with one value. Third line creates a set with multiple values"
+    },
+    "unordered_mutable_list_start_number": {
+      "code": [
+        ""
+      ],
```

Comment:
These empty code blocks for unimplemented concepts can be removed.

Reply:
Will do